client/my-sites/stats/stats-subscribers-chart-section/index.tsx

```diff
@@ -175,16 +175,42 @@ export default function SubscribersChartSection( {
 			{ ! isChartLoading && chartData.length !== 0 && (
 				<>
 					<div className="subscribers-section-legend" ref={ legendRef }></div>
-					<UplotChart
-						data={ chartData }
-						legendContainer={ legendRef }
-						period={ period }
-						// Use variable --studio-jetpack-green for chart colors on Odyssey Stats.
-						mainColor={ isOdysseyStats ? '#069e08' : undefined }
-						fillColorFrom={ isOdysseyStats ? 'rgba(6, 158, 8, 0.4)' : undefined }
-						fillColorTo={ isOdysseyStats ? 'rgba(6, 158, 8, 0)' : undefined }
-						yAxisFilter={ hideFractionNumber }
-					/>
+					{ isChartLibraryEnabled && (
+						<LineChart
+							chartData={ [
+								{
+									label: 'Subscribers',
+									options: {
+										stroke: '#069e08',
+									},
+									data: data?.data?.map( ( point ) => ( {
+										date: new Date( point.period ),
+										value: point.subscribers || 0,
+									} ) ) || [
```

Comment:
Happy to keep the dummy data for now, but let's remove it before going live

Reply:
Yep totally 👍  This is just a small chunk of a PR to put the chart component in. Will definitely replace the dummy data in one of my followup PRs. Just making an adjustment to the component structure to handle both realtime (as currently) data and static/periodic data.